samples/dymaptic.GeoBlazor.Core.Sample.TokenRefresh/dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client/Program.cs

```diff
@@ -0,0 +1,76 @@
+using dymaptic.GeoBlazor.Core;
+using dymaptic.GeoBlazor.Core.Model;
+using dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client;
+using dymaptic.GeoBlazor.Core.Sample.TokenRefresh.Client.Models;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
+using System.Net.Http.Json;
+using System.Text.Json;
+
+var builder = WebAssemblyHostBuilder.CreateDefault(args);
+
+// STEP 1: Retrieve client configuration from server API
+// This approach separates concerns by keeping sensitive credentials (client secrets) on the server
+// while allowing the WebAssembly client to access necessary configuration for GeoBlazor initialization.
+var httpClient = new HttpClient { BaseAddress = new Uri(builder.HostEnvironment.BaseAddress) };
```

Comment:
this client probably should have a `using` and scope (brackets) on it, so it gets disposed after the initial call. Or, alternatively, register this one on line 73 rather than creating a new one.